src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const successMessage = `<div class="message">Promise was resolved!</div>`;
+const errorMessage = `<div class="message error-message">Promise was rejected!</div>`;
+
+const successHandler = () => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', successMessage);
+};
+const errorHandler = () => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', errorMessage);
+};
+
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1.then(successHandler).catch(errorHandler);
```

Comment:
You are attaching both success and error handlers to `promise1`. However, in the creation of `promise1`, there is no possibility of it being rejected, as you only call `resolve`. Therefore, attaching an error handler to `promise1` is unnecessary. Similarly, `promise2` is always rejected, so attaching a success handler to it is not needed.